### PR TITLE
Test comments

### PR DESCRIPTION
**Pull request by nimrodkor**
# Generated description

This pull request updates a comment in the `additions.py` file to improve clarity by changing 'find two solutions' to 'find both solutions'. The change is minor and focuses on enhancing the readability of the code comments.

### Comment Clarification
* [`python/additions.py`](https://github.com/baz-scm/diff-test/pull/3/files#diff-a6f7397fae47cc4fc7c8dcaee5417ec21ef7091ef39e25fca73eee55822fffaf)

### CODE REVIEW

File: python/additions.py [hunk 1]
```diff
@@ -10,14 +10,12 @@
 # calculate the discriminant
 d = (b**2) - (4*a*c)
 
-# find two solutions
+# find both solutions
```
Comment (nimrodkor) [inline python/additions.py:13]: This is one comment
that isn't that short.

Maybe try another one right after

File: python/additions.py [hunk 2]
```diff
@@ -10,14 +10,12 @@
...
 sol1 = (-b-cmath.sqrt(d))/(2*a)
 sol2 = (-b+cmath.sqrt(d))/(2*a)
-
```
Comment (nimrodkor) [inline python/additions.py:16]: Why remove this line? Where's the linter?

File: python/additions.py [hunk 3]
```diff
@@ -10,14 +10,12 @@
...
 print('The solution are {0} and {1}'.format(sol1, sol2))
-
 # Taking kilometers input from the user
-kilometers = float(input("Enter value in kilometers: "))
+kilometers = float(input("Enter distance in kilometers: "))
```
Comment (nimrodkor) [inline python/additions.py:18]: Please revert to using `value`

File: python/additions.py (hunk 2 above)
Comment (nimrodkor) [reply python/additions.py:16]: Checking why linting isn't working

Comment (nimrodkor) [reply python/additions.py:16]: Alright, found the issue, it's in main branch and not on this branch

Comment (nimrodkor) [reply python/additions.py:16]: Fix was pushed, merging master

File: python/additions.py [hunk 4]
```diff
@@ -10,7 +10,7 @@
 # calculate the discriminant
 d = (b**2) - (4*a*c)
 
-# find two solutions
+# find both solutions
```
Comment (michaelAppsforce) [inline python/additions.py:13]:
```suggestion
# find both solutions 
I need to test a long suggestion that overflows sol1 = (-b-cmath.sqrt(d))/(2*a) sol1 = (-b-cmath.sqrt(d))/(2*a)
```

File: python/additions.py [hunk 5]
```diff
@@ -10,7 +10,7 @@
...
 sol1 = (-b-cmath.sqrt(d))/(2*a)
```
Comment (nimrodkor) [inline python/additions.py:14]: <p>Test a very very very very very very very very very very very very very very very long comment</p>

**Pull request closed without merging**